addrmgr/network.go

```diff
@@ -116,18 +116,19 @@ func isOnionCatTor(netIP net.IP) bool {
 	return onionCatNet.Contains(netIP)
 }
 
-// NetworkAddress type is used to classify a network address.
-type NetworkAddress int
+// NetAddressType is used to indicate which network a network address belongs
+// to.
+type NetAddressType int
 
 const (
-	LocalAddress NetworkAddress = iota
+	LocalAddress NetAddressType = iota
```

Comment:
`LocalAddress` can be removed. Since it's only used in `getNetwork` which is called from a single place, the call to `isLocal` can be moved up the stack. In it's place for a zero value, we can use something like `UnknownAddressType` to represent an address whose type cannot be determined.

Reply:
This can be deferred since it's not strictly necessary for the decoupling and the field is not used at this point anyways.